apache/incubator-pegasus
# refactor: move db_get into rocksdb_wrapper

### What problem does this PR solve? 
Add a  class `rocksdb_wrapper` to wrap the operation of rocksdb, and put `db_get` into it. 


### Check List 

Tests 

- Unit test
- Manual test (add detailed scripts or steps below)

```
>>> use temp
OK
>>> set a b 1
OK

app_id          : 2
partition_index : 4
decree          : 1
server          : 10.232.55.210:34802
>>> incr a b 
2

app_id          : 2
partition_index : 4
decree          : 2
server          : 10.232.55.210:34802
>>> check_and_set a -c b -t exist -s b1 -v c1 -r
hash_key: "a"
check_sort_key: "b"
check_type: exist
set_sort_key: "b1"
set_value: "c1"
set_value_ttl_seconds: 0
return_check_value: true

Set succeed.

Check value: "2"

app_id          : 2
partition_index : 4
decree          : 4
server          : 10.232.55.210:34802
>>> get a b1 
"c1"

app_id          : 2
partition_index : 4
server          : 10.232.55.210:34802
>>> set a b c 1
OK

app_id          : 2
partition_index : 4
decree          : 7
server          : 10.232.55.210:34802
>>> check_and_set a -c b -t exist -s b1 -v c1 -r
hash_key: "a"
check_sort_key: "b"
check_type: exist
set_sort_key: "b1"
set_value: "c1"
set_value_ttl_seconds: 0
return_check_value: true

Set failed, because check not passed.

Check value not exist.

app_id          : 2
partition_index : 4
decree          : 8
server          : 10.232.55.210:34802
>>> set a b c
OK

app_id          : 2
partition_index : 4
decree          : 9
server          : 10.232.55.210:34802
>>> check_and_set a -c b_not_exist -t exist -s b1 -v c1 -r
hash_key: "a"
check_sort_key: "b_not_exist"
check_type: exist
set_sort_key: "b1"
set_value: "c1"
set_value_ttl_seconds: 0
return_check_value: true

Set failed, because check not passed.

Check value not exist.

app_id          : 2
partition_index : 4
decree          : 10
server          : 10.232.55.210:34802
```

Related changes

- Need to cherry-pick to the release branch
- Need to be included in the release note
